### DISCUSSION TITLE
Form submission with GET method and `up-target` does not change browser URL

### DISCUSSION THREAD

**Discussion by alexandremjacques** [unpoly/unpoly#657] [Q&A] [answered]
Hi!

I'm using Django and django-filter to make a simple filter for data in a HTML table.

django-filter, by default, expects form submissions to be made with GET method so it can read the URL parameters to render the selected options (all filters are `<select>` tags). One thing I noticed is that if I add an `up-target` to the `form` tag, as Unpoly handles the submit, the URL in the browser is not changed (so, if I reload the page, I don't have the filters applied again).

Is that an expected behaviour?

TIA

**Comment by triskweline** [comment, marked as answer]
Hi @alexandremjacques!

By default Unpoly will only change the URL if the main element is changed. You can force a history change by setting an `[up-history=true]` attribute on your form.

For details, see [when history is changed](https://unpoly.com/updating-history#when-history-is-changed) in the docs.

**Reply by alexandremjacques** [reply]
Thanks! Works perfectly.

So nice that Unpoly has almost everything figured out. :D
